agent/eventhandler/task_handler_types.go

```diff
@@ -200,7 +203,8 @@ func setTaskChangeSent(event *sendableEvent, dataClient data.Client) {
 	}
 	for _, containerStateChange := range event.taskChange.Containers {
 		container := containerStateChange.Container
-		updateContainerSentStatus(container, containerStateChange.Status, containerStateChange.ManagedAgents, dataClient)
+		//updateContainerSentStatus(container, containerStateChange.Status, containerStateChange.ManagedAgents, dataClient)
```

Comment:
are you addressing updating the sent status of managed agents as part of this PR?

Reply:
no that will be a separate PR.  (This is to immediately unblock testing.)